src/validations/rules/sar.sch

```diff
@@ -176,6 +175,140 @@
 
     </sch:pattern>
 
+    <sch:pattern
+        id="results">        
+
+        <sch:rule
+            context="oscal:result">
+            <sch:let
+                name="sap-import-url"
+                value="resolve-uri(/oscal:assessment-results/oscal:import-ap/@href, base-uri())" />
+            <sch:let
+                name="sap-available"
+                value="
+                    if (: this is not a relative reference :) (not(starts-with($sap-import-url, '#')))
```

Comment:
The SAP can be available via an intra-document relative reference which requires using the referenced `resource` to obtain (if possible) the SAP via the `resource/rlink`.